share/OpenMS/SCRIPTS/ropenms.R

```diff
@@ -5,6 +5,8 @@
 ### if not installed:
 ### - install pyopenms (https://pyopenms.readthedocs.io/en/latest/installation.html)
 ###   make sure R is using the same python environment as your pyopenms installation
+###   eg. use_python("/usr/local/miniconda3/envs/py37/bin/python")
+###   or  Sys.setenv(RETICULATE_PYTHON = "/usr/local/miniconda3/envs/py37/bin/python")
```

Comment:
Here it is important to set the PATH before calling the library(reticulate) - if it is done otherwise it won't work.

use_python can be used after calling the library (but does not seem to work in some cases)